packages/react-search-ui/src/containers/PagingInfo.js

```diff
@@ -12,7 +13,8 @@ export class PagingInfoContainer extends Component {
     pagingStart: PropTypes.number.isRequired,
     pagingEnd: PropTypes.number.isRequired,
     resultSearchTerm: PropTypes.string.isRequired,
-    totalResults: PropTypes.number.isRequired
+    totalResults: PropTypes.number.isRequired,
+    results: PropTypes.arrayOf(ResultType)
```

Comment:
This is similar to Sorting.js container (but only relates to `results` prop):
> Sorry if I'm wrong, but why we need to exclude results and searchTerm from the ...rest here?
It looks like for this file we could remove them both from this.props destructuring and from propTypes.
Does it make sense?